src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -112,6 +112,16 @@ public void setPerson(Person target, Person editedPerson) {
         addressBook.setPerson(target, editedPerson);
     }
 
+    @Override
+    public void addMeeting(Person target, Meeting meeting) {
+        target.getMeetings().addMeeting(meeting);
+        addressBook.addMeeting(meeting);
+    }
+
+    public String listMeetings() {
+        return addressBook.listMeetings();
+    }
+
     //=========== Filtered Person List Accessors =============================================================
```

Comment:
Is it not?

Reply:
Notes never said anything, and CI pipeline's checkstyle won't complain, but I think it might be seen as not professional enough, just change to "Filtered person list accessors below" or something